ngraph/core/include/ngraph/op/grn.hpp

```diff
@@ -7,9 +7,7 @@
 #include <memory>
 
 #include "ngraph/node.hpp"
-#include "ngraph/op/util/fused_op.hpp"
-
-NGRAPH_SUPPRESS_DEPRECATED_START
+#include "ngraph/op/op.hpp"
```

Comment:
nit: I think `#include "ngraph/node.hpp"` is not needed anymore.

Reply:
Ok, removed